### PR TITLE
Add prompts from OpenAI Community and personal blogs

### PR DESCRIPTION
**Pull request by AChenQ**

### CODE REVIEW

File: openPrompt/analogies/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+write : book :: paint : picture
+drink : coffee :: eat : chicken
+sleep : bed :: drive : car
+ride : horse :: sail :
+
+```
+
+#### Response
+```
+
```
Comment (linjiX) [inline openPrompt/analogies/readme.md]: Why we have an empty line here? Does it necessary？

File: openPrompt/analogies/readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+### Examples
+
+#### Prompt
+```
+write : book :: paint : picture
+drink : coffee :: eat : chicken
+sleep : bed :: drive : car
+ride : horse :: sail :
+
```
Comment (linjiX) [inline openPrompt/analogies/readme.md]: Does the last empty line here matters?

File: openPrompt/described_structure/readme.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+## described_structure
```
Comment (linjiX) [inline openPrompt/described_structure/readme.md]: I think use snake_case in title is improper，will it be better to use "Title Case" for title?

File: openPrompt/described_structure/readme.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+### Description
+With a described structure there is even less information given, consider the following prompt, now with a described JSON structure
+
+### Examples
+
+#### Prompt1
```
Comment (linjiX) [inline openPrompt/described_structure/readme.md]: shall we use markdown list grammar for multiple prompts?

File: openPrompt/song_generation/prompts.json [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        {
+            "prompt": "\nCHORDS (happy):\n",
+            "response": "\nMy pangol[G]in [Em]heart has [Am]scales of [Am]bone\nmy [Am]forelegs [Em]fur[Em7]ry and [C]black,\nre[C]tractile [Em]elastic [C]spring[G]loaded [C]pincers\nand [G]looming like a [Am]hidden[Am] [Am]gem in the [Em]dusk\n"
+        },
+        {
+            "prompt": "\nLyrics to #1 Chart Topping Hit \"Pangolin\", by Bono and Larry Mullen jr.\n",
+            "response": "\nIntro by Bono:\nNow my right hand man, Larry Mullen\nSings, and you know how we do it\nWith a perfect unisono\nYou'll need a translation when he speak it\nSo, let me break it down for you\nBut you can't break it down for us\nLarry,\nWhat did you see there, what did you feel?\nBeen wanting to ask ya for such a long time.\n"
+        },
+        {
+            "prompt": "",
```
Comment (linjiX) [inline openPrompt/song_generation/prompts.json]: why these prompts are all empty?

File: openPrompt/song_generation/readme.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+And big sharp points at the end of his toes
+Called a Pangolin it's the most treasured animal in all of Africa
+For it's fur and for it's meat it's a delicacy
+But they will make a fine pelt for the lady in my life.
+
+```
+
+
+
+categories: Generation
```
Comment (linjiX) [inline openPrompt/song_generation/readme.md]: shall we add code block grammar for categories?

File: openPrompt/1st_to_3rd_person_writing/prompts.json [hunk 7]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (linjiX) [inline openPrompt/1st_to_3rd_person_writing/prompts.json]: better to add the urls of the source OpenAI Community and personal blogs in the commit message body

**Review by linjiX — CHANGES_REQUESTED** [review]

File: openPrompt/described_structure/readme.md (hunk 4 above)
Comment (AChenQ) [reply openPrompt/described_structure/readme.md]: It seems inappropriate to use list grammar both Prompt and Response, I need some examples

File: openPrompt/english_to_python/readme.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
+## EnglishToPython
```
Comment (linjiX) [inline openPrompt/english_to_python/readme.md]: will it be better to add space between words in the title?

File: openPrompt/analogies/prompts.json [hunk 9]
```diff
@@ -0,0 +1,13 @@
+{
+    "category": [
+        ""
```
Comment (linjiX) [inline openPrompt/analogies/prompts.json]: Should this be an empty list?

File: openPrompt/cover_letter_generator/readme.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+I'm keen to join your technology consulting team because I know I would thrive in the challenging and fast-paced environment. I'd love the opportunity to learn from your expert consultants and to work with a team that is working on a diverse range of projects. I've heard great things about Deloitte and would love to find out more. I look forward to hearing from you.
+
+Yours sincerely,
+Ella Duffy
+```
+
+
+
+```
+categories: Generation
```
Comment (linjiX) [inline openPrompt/cover_letter_generator/readme.md]: will it be better to make "categories" as a subtitle?

File: openPrompt/idea_generator/readme.md [hunk 11]
```diff
@@ -0,0 +1,65 @@
+## IdeaGenerator
+### Description
```
Comment (linjiX) [inline openPrompt/idea_generator/readme.md]: should we remove the description section if it is empty?

File: openPrompt/idea_generator/readme.md [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+A woman has a rare African plant named after her. She is taken prisoner by a rich collector who wants the plant.
+```
+
+
+
+```
+categories: Generation
+```
+
+[source](https://www.notion.so/Idea-Generator-3f5492dd27c44c23a1f1629e8fc03b00)
```
Comment (linjiX) [inline openPrompt/idea_generator/readme.md]: will it be better to make "source" as a subtitle?

File: openPrompt/script_to_novel/readme.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+
+### Examples
+
+#### Prompt
+```
+I was asked to convert this script into a written story:
+
+"""
+DR JANE ELDER
+Where do you think you��re going?
```
Comment (linjiX) [inline openPrompt/script_to_novel/readme.md]: what it the unknown symbol between "you" and "re"

**Review by linjiX — CHANGES_REQUESTED** [review]

File: openPrompt/analogies/prompts.json [hunk 14]
```diff
@@ -0,0 +1,13 @@
+{
+    "category": [
```
Comment (linjiX) [inline openPrompt/analogies/prompts.json]: this field name should be `categories`

**Review by linjiX — CHANGES_REQUESTED** [review]

File: openPrompt/1st_to_3rd_person_writing/readme.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
+## 1St To 3Rd Person Writing
```
Comment (linjiX) [inline openPrompt/1st_to_3rd_person_writing/readme.md]: - `1St` -> `1st`
- `3Rd` -> `3rd`

File: openPrompt/idea_generator/readme.md [hunk 16]
```diff
@@ -0,0 +1,62 @@
+## Idea Generator
+### Categories
+```
+['Generation']
```
Comment (linjiX) [inline openPrompt/idea_generator/readme.md]: The categories format is different with others?

Comment (linjiX) [reply openPrompt/idea_generator/readme.md]: will it be better to use inline code block for categories?
such as:
`category1`, `category2`

**Review by linjiX — CHANGES_REQUESTED** [review]

File: openPrompt/script_to_novel/readme.md [hunk 17]
```diff
@@ -0,0 +1,68 @@
...
+She took a step towards him. Then she raised the gun and fired a shot over his head.
+
+"If you want to walk out of here with anything above your neck I suggest you put the computer down and lay on the floor," Dr. Elder demanded.
+
+
+Dr. Elder took another step toward him. The masked gunman squatted on the floor with the computer on his lap.
+
+Dr. Elder took one step and kicked the computer out of his hands.
+
+"Okay, I��ll play," the masked gunman laughed. "Give me a second while I turn this off."
```
Comment (linjiX) [inline openPrompt/script_to_novel/readme.md]: seems the strange symbol still exist

**Review by linjiX — CHANGES_REQUESTED** [review]

File: openPrompt/english_to_python/readme.md [hunk 18]
```diff
@@ -0,0 +1,47 @@
+## English To Python
```
Comment (linjiX) [inline openPrompt/english_to_python/readme.md]: `To` -> `to`

File: openPrompt/1st_to_3rd_person_writing/prompts.json (hunk 7 above)
Comment (linjiX) [inline openPrompt/1st_to_3rd_person_writing/prompts.json]: the naming style of the root folder "openPrompt" is different with its sub folders
will it be better to `openPrompt` -> `open_prompt(s)`?

**Review by linjiX — CHANGES_REQUESTED** [review]

**Review by linjiX — APPROVED** [review]

**Pull request merged**